src/classes/particle.ts

```diff
@@ -153,9 +159,9 @@ export class Particle {
 
     private calcVelocity(options: IOptions) {
         const velbase = this.getVelBase(options);
-        let res: IVelocity = {
+        let res = {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Identifier 'res' is never reassigned; use 'const' instead of 'let'.](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5011230)